sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientMockTests.cs

```diff
@@ -38,44 +38,66 @@ private TextAnalyticsClient CreateTestClient(HttpPipelineTransport transport)
         [Test]
         public async Task RecognizeEntitiesResultsSorted_NoErrors()
         {
-            var mockResults = new List<RecognizeEntitiesResult>()
-            {
-                TextAnalyticsModelFactory.RecognizeEntitiesResult("1", new TextDocumentStatistics(),
-                    new CategorizedEntityCollection
-                    (
-                        new List<CategorizedEntity>
+            using var Stream = new MemoryStream(Encoding.UTF8.GetBytes(@"
```

Comment:
nittiest nit of all nits: instead of `Stream`, it would be better if we named it `stream` since it's a local variable. The same comment applies to other variables with the same name declared in this test file.

Definitely not going to push on with this one.

Reply:
updated!